### ISSUE TITLE
Parental strain combo box shows many blanks

### ISSUE THREAD

**Issue by vladimirg** [berman-lab/ymap#65]
1. Create a new project.
2. Choose A21-s02-m09-r07 as the reference genome.
3. Click on the "parental strain" combo box.

There are many blanks there. On inspection, these are the demonstration projects from the Ymap paper that are stored in the `default` user. They should probably be filtered out.

**Comment by darrenabbey** [comment]
Agreed. If a user wants to use one of those datasets as a parent in their analysis, it is simple to download them from the "Example Datasets" tab.
